packages/mux-player/src/template.ts

```diff
@@ -41,7 +43,7 @@ const getHotKeys = (props: MuxTemplateProps) => {
 export const content = (props: MuxTemplateProps) => html`
   <media-theme
     template="${props.themeTemplate ?? muxTemplate.content.children[0]}"
-    stream-type="${isLiveOrDVR(props) ? 'live' : 'on-demand'}"
```

Comment:
(blocking): I think we still require this here because when the user manually sets the stream-type on mux-player that one should be used.

Reply:
for sure/good callout. I removed all of the explicit setting until I resolved all of the assumptions so everything could work with the inferred case. With the new architecture/latest push, I'm solving it slightly differently by overriding it on `<mux-video>` and letting the values "flow" through media chrome. I have, however, added a `default-stream-type` to correspond with overriding the newly added corresponding attribute in `<media-controller>`.